engine/gamesys/src/gamesys/components/comp_label.cpp

```diff
@@ -257,6 +257,22 @@ namespace dmGameSystem
         return dmGameObject::CREATE_RESULT_OK;
     }
 
+    Matrix4 CompLabelLocalTransform(
+        const Point3& position,
+        const Quat& rotation,
+        const Vector3& scale,
+        const Vector3& size,
+        uint32_t pivot)
```

Comment:
We don't generally put each argument onto a separate line.
Best is to follow the style of the rest of the file one is editing.

Reply:
Changed codestyle here. Also I did some tests refactoring (please take a look).